### ISSUE TITLE
【论文阅读】Batch Normalization：Accelerating Deep Network Training by Reducing Internal Covariate Shift - welcome to x-jeff blog

### ISSUE THREAD

**Issue by x-jeff** [x-jeff/GitTalk#229]
http://shichaoxin.com/2021/11/02/%E8%AE%BA%E6%96%87%E9%98%85%E8%AF%BB-Batch-Normalization-Accelerating-Deep-Network-Training-by-Reducing-Internal-Covariate-Shift/ 

 xinshichao's blog
